### DISCUSSION TITLE
Forge install not working for installing dependency

### DISCUSSION THREAD

**Discussion by Aamirusmani1552** [foundry-rs/foundry#4565] [Help]
hello, 
I am trying to install `Openzeppelin/openzeppelin-contracts` with `forge intall` but it is giving me this weird error.
```bash
PS F:\P2P\P2PBackendSoldity> forge install OpenZeppelin/openzeppelin-contracts
Installing openzeppelin-contracts in "F:/P2P/P2PBackendSoldity/lib\\openzeppelin-contracts" (url: Some("https://github.com/OpenZeppelin/openzeppelin-contracts"), tag: None)
Error:
The target directory is a part of or on its own an already initialized git repository,
and it requires clean working and staging areas, including no untracked files.

Check the current git repository's status with `git status`.
Then, you can track files with `git add ...` and then commit them with `git commit`,
ignore them in the `.gitignore` file, or run this command again with the `--no-commit` flag.

If none of the previous steps worked, please open an issue at:
https://github.com/foundry-rs/foundry/issues/new/choose
```

I recently started learning foundry so i don't have much idea about the error. please help

**Comment by cr00k3d3y3** [comment]
Go figure that it is the exact error I am getting and there are no answers.  Real valuable discussion forum

**Comment by DaniPopes** [comment]
Hello sgilbert79, have you tried reading the error message and going through the listed steps?

**Comment by cr00k3d3y3** [comment]
Yep

**Comment by cr00k3d3y3** [comment]
I guess I just have to get with the fact that I'll never understand this
stuff and I'll never find the help I need when I don't. So I guess it's
just God's way of telling me to leave it alone

**Comment by Vicorico17** [comment]
I had the same issue and this is how I solved it.

I ran 

```
git rev-parse --show-toplevel
```

to see where what git was my directory part of

after I got the answer I found out I had a git repo from my home directory (weird) so I ran

```
rm -rf /XXX/.git ( change the XXX with your response from git rev-parse --show-toplevel)
```

hope this helps
